Storage/SQL/SQLitePasswords.swift

```diff
@@ -34,9 +38,20 @@ private class PasswordsTable<T>: GenericTable<Password> {
         args.append(item.timeCreated)
         args.append(item.timeLastUsed)
         args.append(item.timePasswordChanged)
+
         args.append(item.username)
-        args.append(item.password)
-        return ("INSERT INTO \(name) (hostname, httpRealm, formSubmitUrl, usernameField, passwordField, guid, timeCreated, timeLastUsed, timePasswordChanged, username, password) VALUES (?,?,?,?,?,?,?,?,?,?,?)", args)
+        switch (encryption) {
+        case EncryptionType.AES256:
+            let secret = SQLitePasswords.secretKey!
```

Comment:
This will crash if secretKey fails. Which I guess is possible since SecRandomCopyBytes can return nil or an error.